player/lua/ytdl_hook.lua

```diff
@@ -142,6 +142,45 @@ local function is_blacklisted(url)
     return false
 end
 
+local function parse_yt_playlist(url, json)
```

Comment:
Should probably include some more detail about the issue or feature this is addressing in the commit message.